host/CMakeLists.txt

```diff
@@ -488,3 +488,321 @@ install(TARGETS oehostapp EXPORT openenclave-targets)
+#  set_source_files_properties(oemr
+#    sgx/enter.c
+#    PROPERTIES
+#    COMPILE_FLAGS "-O2 -fno-omit-frame-pointer")
+#endif()
+
+#add_dependencies(oemr syscall_untrusted_edl)
+#add_dependencies(oemr tee_untrusted_edl)
+if(OE_SGX)
+  add_dependencies(oemr sgx_untrusted_edl switchless_untrusted_edl)
+endif()
+
+# TODO: Replace these with `find_package` and add as dependencies to
+# the CMake package.
+if (UNIX)
+	if (NOT TARGET openenclave::crypto)
+	  find_library(CRYPTO_LIB NAMES crypto)
+	  if (NOT CRYPTO_LIB)
+	    message(FATAL_ERROR "-- Looking for crypto library - not found")
+	  else ()
+	    message("-- Looking for crypto library - found")
+	    add_library(openenclave::crypto SHARED IMPORTED)
+	    set_target_properties(openenclave::crypto PROPERTIES IMPORTED_LOCATION ${CRYPTO_LIB})
+	  endif ()
+	endif ()
+
+    if (NOT TARGET openenclave::dl)
+      find_library(DL_LIB NAMES dl)
+      if (NOT DL_LIB)
+        message(FATAL_ERROR "-- Looking for dl library - not found")
+      else ()
+        message("-- Looking for dl library - found")
+        add_library(openenclave::dl SHARED IMPORTED)
+        set_target_properties(openenclave::dl PROPERTIES IMPORTED_LOCATION ${DL_LIB})
+      endif ()
+    endif ()
+endif ()
+
+find_package(Threads REQUIRED)
+
+if (UNIX)
+	target_link_libraries(oemr PRIVATE
+	    openenclave::crypto
+		#	    openenclave::dl
+		Threads::Threads)
+  if (OE_TRUSTZONE)
+    target_include_directories(oemr PRIVATE
+      ${OE_TZ_OPTEE_CLIENT_INC})
+    target_link_libraries(oemr PRIVATE teec)
+  endif ()
+elseif (WIN32)
+  target_include_directories(oemr PRIVATE
+    ${CMAKE_SOURCE_DIR}/3rdparty/mbedtls/mbedtls/include)
+endif ()
+
+# For including edge routines.
+target_include_directories(oemr PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
+
+# Compile definitions and options
+target_compile_definitions(oemr
+  PUBLIC
+  # NOTE: This definition is public to the rest of our project's
+  # targets, but should not yet be exposed to consumers of our
+  # package.
+  $<BUILD_INTERFACE:OE_API_VERSION=2>
+  PRIVATE
+  OE_BUILD_UNTRUSTED
+  OE_REPO_BRANCH_NAME="${GIT_BRANCH}"
+  OE_REPO_LAST_COMMIT="${GIT_COMMIT}")
+
+if (USE_DEBUG_MALLOC)
+  target_compile_definitions(oemr PRIVATE OE_USE_DEBUG_MALLOC)
+endif ()
+
+if (UNIX)
+  target_compile_options(oemr
+     PRIVATE -Wno-attributes -Wmissing-prototypes -fPIC ${PLATFORM_FLAGS}
+     PUBLIC -fstack-protector-strong)
+  target_compile_definitions(oemr
+     PRIVATE _GNU_SOURCE
+     PUBLIC $<$<NOT:$<CONFIG:debug>>:_FORTIFY_SOURCE=2>)
+endif ()
+
+if (CMAKE_C_COMPILER_ID MATCHES GNU)
+  target_compile_options(oemr PRIVATE -Wjump-misses-init)
+endif ()
+
+target_compile_options(oemr PRIVATE -DOEMR)
+set_property(TARGET oemr PROPERTY
+  ARCHIVE_OUTPUT_DIRECTORY ${OE_LIBDIR}/openenclave/host)
+
+if (UNIX)
+  target_link_libraries(oemr INTERFACE
+      -Wl,-z,noexecstack)
+endif ()
+
+# Install targets
+install(TARGETS oemr EXPORT openenclave-targets
```

Comment:
Since oems is used only to build oesign, and is not intended to be used by OE SDK user, we need not install it.

Reply:
Good point, this is just a internal and intermediate library.